src/main/java/org/ibp/api/rest/study/StudyEntryResource.java

```diff
@@ -143,4 +145,23 @@ public long getCount() {
 
 		return new ResponseEntity<>(entryDescriptors, HttpStatus.OK);
 	}
+
+	@ApiOperation(value = "Get Study Test Entries count",
+		notes = "Get Study Test Entries count")
+	@PreAuthorize("hasAnyAuthority('ADMIN','STUDIES','MANAGE_STUDIES')")
+	@RequestMapping(value = "/{crop}/programs/{programUUID}/studies/{studyId}/entries/count-test-entries", method = RequestMethod.GET)
```

Comment:
It is not clear to me why we need these 2 services. Should not be better just to add this information in the header response of the GET entries? it is metadata of the GET service right? 
If that is not possible I think a better option is to create /entries/metadata service and return all the counts inside, will be more REST and BMSAPI compliant 
@darla-leafnode @mderamos